tests/common/mock_cgroup_commands.py

```diff
@@ -67,23 +71,28 @@
 
 ]
 
-__service_file = "{0}.service".format(get_osutil().get_service_name())
-
-__MOCKED_FILES = [
-    (r"^/proc/self/cgroup$", os.path.join(data_dir, 'cgroups', 'proc_self_cgroup')),
-    (r"^/proc/[0-9]+/cgroup$", os.path.join(data_dir, 'cgroups', 'proc_pid_cgroup')),
-    (r"^/sys/fs/cgroup/unified/cgroup.controllers$", os.path.join(data_dir, 'cgroups', 'sys_fs_cgroup_unified_cgroup.controllers')),
-    (r"^/lib/systemd/system/{0}$".format(__service_file), os.path.join(data_dir, 'init', __service_file + "_system-slice")),
+_MOCKED_FILES = [
+    ("/proc/self/cgroup", os.path.join(data_dir, 'cgroups', 'proc_self_cgroup')),
+    (r"/proc/[0-9]+/cgroup", os.path.join(data_dir, 'cgroups', 'proc_pid_cgroup')),
+    ("/sys/fs/cgroup/unified/cgroup.controllers", os.path.join(data_dir, 'cgroups', 'sys_fs_cgroup_unified_cgroup.controllers'))
 ]
 
-__MOCKED_PATHS = [
+_MOCKED_PATHS = [
     r"^(/lib/systemd/system)",
     r"^(/etc/systemd/system)"
 ]
 
-# W0212: Access to a protected member __commands of a client class (protected-access) - Disabled: patcher.__commands is added
-# only for debugging purposes and should not be public (hence it is marked as private).
-# pylint: disable=protected-access
+
+#  Class has no __init__ method (no-init): Disabled; the class is just an enum
+class UnitFilePaths:  # pylint: disable=no-init
```

Comment:
In favor with removing this pylint too globally?

Reply:
sounds good; will submit a PR